packages/edit-site/src/store/actions.js

```diff
@@ -257,22 +257,48 @@ export function setEditedPostContext( context ) {
 export const setPage =
 	( page ) =>
 	async ( { dispatch, registry } ) => {
-		if ( ! page.path && page.context?.postId ) {
-			const entity = await registry
+		let template;
+		const getDefaultTemplate = async ( slug ) =>
```

Comment:
It's a bit mind bending that [here](https://github.com/WordPress/gutenberg/pull/51477/files#diff-345eaac452a15b557fde376f844090e4981ae5781716bf2acae186c87d63508aR25) the default template is an empty string and yet we still "get" it. Should it be a boolean like hasDefaultTemplate somewhere?

Reply:
This is core WP here..